tasks/composer.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: PHP | Check composer version
+  command: "composer --version"
+  register: php_composer_check
+  changed_when: false
+  ignore_errors: true
+
+- name: PHP | Download composer installer
+  get_url:
+    url: "{{ php_composer_installer_url }}"
+    dest: "{{ php_composer_installer_path }}"
+  when: php_composer_reinstall or php_composer_check is failed or php_composer_version not in php_composer_check.stdout
+
+- name: Get and verify composer checksum
+  block:
+    - name: PHP | Get composer installer checksum
+      stat:
+        path: "{{ php_composer_installer_path }}"
+        checksum_algorithm: sha384
+        get_checksum: yes
+      register: installer_stat
+
+    - name: PHP | Verify composer installer checksum
+      fail:
+        msg: "Failure, composer installer is not correct."
+      when: installer_stat.stat.checksum != '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3'
```

Comment:
This hash changes in each composer release. It must be put in a var.

Reply:
done :)